include/clang/Sema/Sema.h

```diff
@@ -9421,11 +9438,13 @@ class Sema {
       bool DiagnoseCFAudited = false, bool ConvertRHS = true);
 
 private:
-  QualType GetCheckedCInteropType(const ValueDecl *Decl);
+  QualType GetCheckedCInteropType(const DeclaratorDecl *Decl);
 public:
-  /// \brief Get the bounds-safe interface type for Entity.
-  /// Returns a null QualType if there isn't one.
-  QualType GetCheckedCInteropType(const InitializedEntity &Entity);
+  /// \brief Given a value with type Ty and bounds Bounds,
+  /// compute the bounds-safe interface type.
+  QualType Sema::GetCheckedCInteropType(QualType Ty,
```

Comment:
This gives a compiler error on my machine, you don't need the `Sema::` prefix as you're already in that scope.

```
/path/to/checkedc-llvm/tools/clang/include/clang/Sema/Sema.h:9445:18: error: extra qualification on member 'GetCheckedCInteropType'
  QualType Sema::GetCheckedCInteropType(QualType Ty,
           ~~~~~~^
1 error generated.
```

Reply:
Fixed - thanks for pointing this out.